docs/en/seatunnel-engine/logging.md

```diff
@@ -80,6 +80,28 @@ appender.file.layout.pattern = [%X{ST-JID}] %d{yyyy-MM-dd HH:mm:ss,SSS} %-5p [%-
 
 SeaTunnel Engine automatically integrates Log framework bridge, allowing existing applications that work against Log4j1/Logback classes to continue working.
 
+### SeaTunnel Log Configuration
+
+### Scheduled Deletion of Old Logs
+
+SeaTunnel supports scheduled deletion of old log files to prevent disk space exhaustion. You can add the following configuration in the `log4j2.properties` file:
+
+```yaml
+    telemetry:
+      logs:
```

Comment:
This is different from the default value of seatunnel.yaml.

![image](https://github.com/user-attachments/assets/5a610fd1-abc6-4cdc-a4dc-889a11037dff)

Reply:
get